### ISSUE TITLE
Fintech api docs and notes

### ISSUE THREAD

**Issue by carlosj934** [Corona-HomeLab/FinSight#9]
Going to leave notes and links to docs for various fintech apps that allow for api access and potential ideas for what I can do with them.

**Comment by carlosj934** [comment]
ADP api docs for payroll management: https://developers.adp.com/build/guides/product-integration-guides?filterQuery=&product=&useCaseTags=Payroll-Management

**Comment by carlosj934** [comment]
Plaid is a financial technology (fintech) company that provides a platform enabling applications to connect with users' bank accounts, credit cards, and other financial accounts securely. Its primary focus is to facilitate access to financial data, allowing developers to create apps and services that integrate seamlessly with financial institutions.

Docs:

- [Plaid Sandbox docs](https://plaid.com/docs/sandbox/)
- [Plaid Pattern app gh](https://github.com/plaid/pattern?tab=readme-ov-file)
  - basically what I was going to do with creating the budgeting app that pulls user bank account info using plaid
  - api endpoints for plaid pattern: https://pastes.dev/5jxToo9QcN
- [anti money laundering docs](https://plaid.com/docs/monitor/)

**Comment by carlosj934** [comment]
Accounting specific software apps and company docs

**Comment by carlosj934** [comment]
> Accounting specific software apps and company docs

Xero developer docs

Xero is a cloud-based small business accounting software designed for startups, growing companies, and established companies. It automates bookkeeping and accounting, providing a range of features to manage business finances.

- https://developer.xero.com/
- https://github.com/XeroAPI
- Xero Open API gh repo
  - https://github.com/XeroAPI/Xero-OpenAPI

**Comment by carlosj934** [comment]
> Accounting specific software apps and company docs

Intuit Developer docs

- https://developer.intuit.com/app/developer/homepage

**Comment by carlosj934** [comment]
Visa Developer docs and articles:

- https://developer.visa.com/
